README.md

```diff
@@ -205,7 +206,7 @@ So to get a pretty kitten at 250 pixels wide, with 50% compression, you would wr
 `<img src="http://imgs.kitty.com/upload/w_250,q_50/https://my.storage.io/imgs/pretty-kitten.jpg">`
 
 
-## Basic Option details
+## Basic Options details
```

Comment:
Either "details of options" or "option details" no  plural adjective for a plurality noun.